### PR TITLE
fix: Repair fat placements

### LINKED ISSUE #8396: The placement of mono and polyunsaturated fat is wrong
### What

The placement of mono and polyunsaturated fat is wrong

### To Reproduce

https://world.openfoodfacts.org/product/0073210000139/extra-virgin-olive-oil-star

### Expected behavior

Should appear under a heading `Unsaturated fat`

### Screenshots

Screenshot 2023-05-04 at 15 00 33

### PR DESCRIPTION
**Pull request by aleene**
### What
Repaired the placement on mon- and polyunsaturated fats in the nutritional table

### Related issue(s) and discussion
- Fixes #8396 and #8395

### CODE REVIEW

**Comment by stephanegigandet** [comment]
@aleene We currently support only 3 levels. We could add more levels, it just needs some changes in the code and the templates.

But it won't solve all issues. Look at this product for instance: https://world.openfoodfacts.org/product/3366321051983/omega-3-doux-st-hubert

image

The table in Food.pm is currently used to set up the order of the nutrients, and their level. But we currently do not use it to show nutrients for which we don't have values, but for which we have values for sub-nutrients.

So for this product, we miss "insaturated fat", so we will show something that looks like:

- Saturated fat
-- Mono insatured fat
-- Poly insaturated fat

Which will look like if mono/poly insaturated fat is part of saturated fat... 

So we will need more changes in the code, to display en empty line for insaturated fat even though we don't have values for it.

**Comment by aleene** [comment]
OK, we can make a separate issue for that.

**Comment by aleene** [comment]
See also: https://nl.openfoodfacts.org/product/8718907341233/light-albert-heijn

**Comment by alexgarel** [comment]
I tested, I got this:
image

File: lib/ProductOpener/Food.pm [hunk 1]
```diff
@@ -500,56 +500,56 @@ It is a list of nutrients names with eventual prefixes and suffixes:
 			'--arachidic-acid-', '--behenic-acid-',
 			'--lignoceric-acid-', '--cerotic-acid-',
 			'--montanic-acid-', '--melissic-acid-',
-			'-monounsaturated-fat-', '-polyunsaturated-fat-',
-			'-omega-3-fat-', '--alpha-linolenic-acid-',
-			'--eicosapentaenoic-acid-', '--docosahexaenoic-acid-',
-			'-omega-6-fat-', '--linoleic-acid-',
+			'-unsaturated-fat-', '--monounsaturated-fat-',
+			'---omega-9-fat-', '--polyunsaturated-fat-',
+			'---omega-3-fat-', '---omega-6-fat-',
```
Comment (alexgarel) [inline lib/ProductOpener/Food.pm:505]: the fact to have four dashes does not harm however, it's displayed as 3 :-)

**Review by alexgarel — APPROVED** [review]
LGTM

File: lib/ProductOpener/Food.pm (hunk 1 above)
Comment (alexgarel) [reply lib/ProductOpener/Food.pm:505]: image

**Pull request merged**